x-pack/plugins/event_log/server/plugin.test.ts

```diff
@@ -42,7 +45,7 @@ describe('event_log plugin', () => {
     const plugin = new Plugin(initializerContext);
     const spaces = spacesMock.createStart();
     // serverless setup is currently empty, and there is no mock
```

Comment:
3 / 3 - Could you remove this comment since it's obviously no longer correct :-)